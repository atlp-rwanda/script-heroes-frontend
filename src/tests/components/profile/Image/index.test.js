import React from 'react';
import { shallow } from 'enzyme';

import ProfileImage from '../../../../components/profile/Image';

describe('<ProfileImage/>', () => {
  it('Should render profile image', () => {
    const component = shallow(<ProfileImage />);
    expect(component).toMatchSnapshot();
  });
});
