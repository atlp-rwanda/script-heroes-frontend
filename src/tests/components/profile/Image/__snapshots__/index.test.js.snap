// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileImage/> Should render profile image 1`] = `
<img
  alt="profile-image"
  className="col-md-6"
  src="test-file-stub"
/>
`;
